R/fileValidation.r

```diff
@@ -189,18 +189,18 @@ file_separator <- function(data, meta) {
     if ((grepl(",", data) == FALSE) && (grepl(",", meta) == FALSE)) {
       output <- list(
         "result" = "FAIL",
-        "message" = paste0("Neither of the data and metadata files appear to be using a comma delimited structure. <br> - See the ", "<a href='https://rsconnect/rsc/stats-production-guidance/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
+        "message" = paste0("Neither of the data and metadata files appear to be using a comma delimited structure. <br> - See the ", "<a href='https://dfe-analytical-services.github.io/analysts-guide/statistics-production/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
       )
     } else {
       if (grepl(",", meta)) {
         output <- list(
           "result" = "FAIL",
-          "message" = paste0("The data file does not appear to be using a comma delimited structure. <br> - See the ", "<a href='https://rsconnect/rsc/stats-production-guidance/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
+          "message" = paste0("The data file does not appear to be using a comma delimited structure. <br> - See the ", "<a href='https://dfe-analytical-services.github.io/analysts-guide/statistics-production/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
```

Comment:
Somewhat outside the scope of this PR but I don't feel like "comma delimited structure" is well enough explained in the analysts guide page - I know what it means but not sure everyone will and I think we could do with an actual example of what different delimiters are and how they're used. Doesn't have to be excessive but could help people spot incorrect formats more quickly if they're unfamiliar with the concept

Reply:
That's fair, for this I've made sure to link to both that section and the part for saving as UTF-8 as that at least gives instructions on how to save files in the right format. Also raised https://github.com/dfe-analytical-services/analysts-guide/issues/69 on the analysts guide for someone to pick up separately to this PR